src/pretix/plugins/stripe/payment.py

```diff
@@ -183,6 +183,13 @@ def settings_form_fields(self):
                      label=_('Credit card payments'),
                      required=False,
                  )),
+                ('method_sepa_debit',
+                 forms.BooleanField(
+                     label=_('SEPA Direct Debit'),
+                     disabled=self.event.currency != 'EUR',
+                     help_text=_('Needs to be enabled in your Stripe account first.'),
+                     required=False,
+                 )),
```

Comment:
Similarly to SOFORT; I think we need to add a very strong warning about the fact that stripe will not immediately mark transactions as complete, but only DAYS later. This is highly unexpected for SEPA debits and will likely confuse lots of users.

Reply:
OK, will add this, we also need to add to enable Stripe webhooks in the Stripe account properly, otherwise you'll never get the actual message that the charge succeeded.